### PR TITLE
Dist plot warning fix

### PR DESCRIPTION
**Pull request by marisand**

### CODE REVIEW

File: OlinkAnalyze/R/dist_plot.R [hunk 1]
```diff
@@ -58,21 +58,13 @@ olink_dist_plot <- function(df, color_g = 'QC_Warning', ...) {
...
-    dplyr::mutate(QC_Warning = dplyr::if_else(QC_Warning == "WARN",
-                                       "Warning",
-                                       QC_Warning)) %>%
-    dplyr::mutate(QC_Warning = dplyr::if_else(QC_Warning == "PASS",
-                                       "Pass",
-                                       QC_Warning)) %>%
-    dplyr::mutate(QC_Warning = dplyr::if_else(QC_Warning == "FAIL",
-                                       "Fail",
-                                       QC_Warning)) %>%
+    dplyr::mutate(QC_Warning = dplyr::if_else(all(toupper(QC_Warning) == 'PASS'),
```
Comment (AskPascal) [inline OlinkAnalyze/R/dist_plot.R:65]: To me aesthetically it looks better with Warning vs WARN

Code wise both work equally well i'd say...

File: OlinkAnalyze/R/dist_plot.R [hunk 2]
```diff
@@ -58,21 +58,13 @@ olink_dist_plot <- function(df, color_g = 'QC_Warning', ...) {
...
+                                              'PASS',
```
Comment (AskPascal) [inline OlinkAnalyze/R/dist_plot.R:66]:
```suggestion
                                              'Pass',
```

File: OlinkAnalyze/R/dist_plot.R [hunk 3]
```diff
@@ -58,21 +58,13 @@ olink_dist_plot <- function(df, color_g = 'QC_Warning', ...) {
...
+                                              'WARNING')) %>%
```
Comment (AskPascal) [inline OlinkAnalyze/R/dist_plot.R:67]:
```suggestion
                                              'Warning')) %>%
```

File: OlinkAnalyze/R/dist_plot.R (hunk 1 above)
Comment (marisand) [reply OlinkAnalyze/R/dist_plot.R:65]: I agree, I was thinking about this as well, but then didn't loop back.

**Pull request merged**